examples/multiple-results/multiple-results.go

```diff
@@ -108,13 +108,19 @@ func multiValueWasmFunctions(ctx context.Context, r wazero.Runtime) (api.Module,
 //go:embed testdata/multi_value_imported.wasm
 var multiValueFromImportedHostWasm []byte
 
+// multiValueWasmFunctions return the WebAssembly which imports the Go-defined "get_age" function which
+// returns multiple results. The source is in testdata/multi_value_imported.wat
```

Comment:
which .. which ... :D

Reply:
@codefromthecrypt need your editorial fix :D

Reply:
"multiValueWasmFunctions returns a wasm-defined module that imports a host function. The host module is defined using wazero.ModuleBuilder, and the wasm module with multiValueFromImportedHostWasm."

the note about the source of multiValueFromImportedHostWasm should be on multiValueFromImportedHostWasm, not here.